src/Google.Bigquery.V2/Preconditions.cs

```diff
@@ -0,0 +1,168 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Nope, because it's internal. See the comment before the namespace declaration :)